docs/victoriatraces/README.md

```diff
@@ -0,0 +1,112 @@
+VictoriaTraces is open source user-friendly database for distributed tracing data 
+from [VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics/).
+
+VictoriaTraces provides the following features:
+- It is resource-efficient and fast. It uses up to [**3.7x less RAM and up to 2.6x less CPU**](https://victoriametrics.com/blog/dev-note-distributed-tracing-with-victorialogs/) than other solutions such as Grafana Tempo.
+- VictoriaTraces' capacity and performance scales linearly with the available resources (CPU, RAM, disk IO, disk space).
+- It accepts trace spans in the popular [OpenTelemetry protocol](https://opentelemetry.io/docs/specs/otel/protocol/)(OTLP).
+- It provides [Jaeger Query Service JSON APIs](https://www.jaegertracing.io/docs/2.6/apis/#internal-http-json) 
+  to integrate with [Grafana](https://grafana.com/docs/grafana/latest/datasources/jaeger/) or [Jaeger Frontend](https://www.jaegertracing.io/docs/2.6/frontend-ui/).
+
+## How does it work
+
+VictoriaTraces is built on top of [VictoriaLogs](https://docs.victoriametrics.com/victorialogs/), which is a log database. 
```

Comment:
I am not sure we should mention it because of the upcoming separation and changes in the storage.

Reply:
Also, I think this section should be afterthe  quick start.